rosetta/errors/details.go

```diff
@@ -14,32 +14,39 @@
 
 package errors
 
+// Detail function can be used to provide more detailed information about a specific
+// error instance.
```

Comment:
// Detail represents a function that can be used to provide more detailed information
// about a specific error instance.